### DISCUSSION TITLE
Blurred image retrieve from "camera.rgb" when replaying.

### DISCUSSION THREAD

**Discussion by joeking11829** [carla-simulator/carla#4825] [Q&A]
Hi Everyone,

CARLA version: 0.9.12
Platform/OS: Ubuntu 18.04

I want to retrieve data from camera.rgb when replay the record.
the rgb images are blurred.

![image](https://user-images.githubusercontent.com/5969040/139810860-b21fb1f4-1fd2-4487-b4bc-7c914e191847.png)

Every images retrieve from replaying have same problem.
The moving vehicles produce blurred image.

Any idea to fix that ?
Thanks.

**Comment by joao-paulo-alves** [comment]
Having the same problem
